cogs/modmail.py

```diff
@@ -718,17 +757,30 @@ def format_log_embeds(self, logs, avatar_url):
         mention = user.mention if hasattr(user, 'mention') else f'`{user.id}`'
 
         if str(user.id) in self.bot.blocked_users:
+            msg = self.bot.blocked_users.get(str(user.id))
+            if msg is None:
+                msg = ''
             del self.bot.config.blocked[str(user.id)]
             await self.bot.config.update()
-            embed = discord.Embed(
-                title='Success',
-                color=self.bot.main_color,
-                description=f'{mention} is no longer blocked'
-            )
+
+            if msg.startswith('System Message: '):
+                reason = msg[16:].strip('. \n\r')
```

Comment:
Add a (python) comment here explaining the strip 🤔

Reply:
Meant to remove trailing spaces and periods, no need for comment, changed it to `msg.strip().rstrip('.')`.